feat(tree-select): add check strategy in tree-select (#758)
TuSimple/naive-ui: src/tree-select/demos/enUS/index.demo-entry.md

| --- | --- | --- | --- |
| cascade | `boolean` | `false` | Whether to do cascade check when use checkboxes. |
| checkable | `boolean` | `false` | Whether to use checkbox to select value. |

> | check-strategy | `string` | `'all'` | set Check Strategy to show parent node or children. all: show all checked node; parent: show all checked parent node when all child node are checked; child: show all child node |

Reviewer: ```suggestion
| check-strategy | `string` | `'all'` | The way to show checked options. `all`: Show all checked node. `parent`: Show all checked parent node when all child node are checked. `child`: show all child node. |
```

中文也顺带改一下
